tests/fireflyClient.test.ts

```diff
@@ -1339,3 +1512,506 @@ describe("FireflyClient", () => {
     });
   });
 });
+
+describe("FireflyClient via ReadOnlyToken", () => {
```

Comment:
These tests are all happy cases of get route, can we add few post route tests?

Reply:
I tried to add post APIs as well upon your suggestion, but those APIs before making an HTTP post call, require more data that we do not have when we initialize readOnly client. ex. walletAddress etc, so the API fails with an error that does not directly related with the feature. I have however, added failure case for post route using readOnlyToken on dapi side to validate the forbidden status it expects.